Form/Form.php

```diff
@@ -236,14 +236,19 @@ protected function getField(string $field)
      */
     public function render()
     {
+
+
         # Build a new panel if variable panels is empty
         if (empty($this->panels)) {
             $panel = $this->panel();
             $this->panels[] = $panel;
         }
 
-        $formTemplate = $this->template ?? config("form.templates.form");
-
+        if (request()->get('customize', 0) == 1) {
+            $formTemplate = config("form.templates.customize");
+        } else {
+            $formTemplate = $this->template ?? config("form.templates.form");
+        }
```

Comment:
trocar para

$formTemplate = $this->template ?? config("form.templates.form");
if (request()->get('customize', 0) == 1) {
    $formTemplate = config("form.templates.customize");
}